workflow/tests/ACCA_Examples/Walker_Residence.xml

```diff
@@ -72,65 +116,638 @@
+            <AttachedToSpace idref='bathroom_1'/>
+            <ExteriorAdjacentTo>outside</ExteriorAdjacentTo>
+            <InteriorAdjacentTo>conditioned space</InteriorAdjacentTo>
+            <WallType>
+              <ConcreteMasonryUnit/>
+            </WallType>
+            <Area>72.0</Area>
+            <Orientation>north</Orientation>
+            <Siding>stucco</Siding>
+            <Insulation>
+              <SystemIdentifier id='WallBathroom1NorthInsulation'/>
+              <AssemblyEffectiveRValue>14.5</AssemblyEffectiveRValue>
+            </Insulation>
+          </Wall>
+          <Wall>
+            <SystemIdentifier id='WallBathroom2North'/>
+            <AttachedToSpace idref='bathroom_2'/>
+            <ExteriorAdjacentTo>outside</ExteriorAdjacentTo>
+            <InteriorAdjacentTo>conditioned space</InteriorAdjacentTo>
+            <WallType>
+              <ConcreteMasonryUnit/>
+            </WallType>
+            <Area>40.0</Area>
+            <Orientation>north</Orientation>
+            <Siding>stucco</Siding>
+            <Insulation>
+              <SystemIdentifier id='WallBathroom2NorthInsulation'/>
+              <AssemblyEffectiveRValue>14.5</AssemblyEffectiveRValue>
+            </Insulation>
+          </Wall>
+          <Wall>
+            <SystemIdentifier id='WallEntryNorth'/>
+            <AttachedToSpace idref='entry'/>
+            <ExteriorAdjacentTo>outside</ExteriorAdjacentTo>
+            <InteriorAdjacentTo>conditioned space</InteriorAdjacentTo>
+            <WallType>
+              <ConcreteMasonryUnit/>
+            </WallType>
+            <Area>56.0</Area>
+            <Orientation>north</Orientation>
+            <Siding>stucco</Siding>
+            <Insulation>
+              <SystemIdentifier id='WallEntryNorthInsulation'/>
+              <AssemblyEffectiveRValue>14.5</AssemblyEffectiveRValue>
+            </Insulation>
+          </Wall>
+          <Wall>
+            <SystemIdentifier id='WallLivingDiningSouth'/>
+            <AttachedToSpace idref='living_dining'/>
+            <ExteriorAdjacentTo>outside</ExteriorAdjacentTo>
+            <InteriorAdjacentTo>conditioned space</InteriorAdjacentTo>
+            <WallType>
+              <ConcreteMasonryUnit/>
+            </WallType>
+            <Area>224.0</Area>
+            <Orientation>south</Orientation>
+            <Siding>stucco</Siding>
+            <Insulation>
+              <SystemIdentifier id='WallLivingDiningSouthInsulation'/>
+              <AssemblyEffectiveRValue>14.5</AssemblyEffectiveRValue>
+            </Insulation>
+          </Wall>
+          <Wall>
+            <SystemIdentifier id='WallLivingDiningWest'/>
+            <AttachedToSpace idref='living_dining'/>
             <ExteriorAdjacentTo>outside</ExteriorAdjacentTo>
             <InteriorAdjacentTo>conditioned space</InteriorAdjacentTo>
             <WallType>
               <ConcreteMasonryUnit/>
             </WallType>
-            <Area>1165.0</Area>
```

Comment:
This wall area was read from the J1 form which is the net area, so the new room by room Walls may yield diffs in block load reports by specifying gross areas.